lib/govuk_publishing_components/presenters/page_with_step_by_step_navigation.rb

```diff
@@ -3,9 +3,10 @@ module Presenters
     # @private
     # Only used by the step by step component
     class PageWithStepByStepNavigation
-      def initialize(content_store_response, current_path)
+      def initialize(content_store_response, current_path, active_step_nav_content_id = false)
         @content_item = content_store_response.to_h
         @current_path = current_path
+        @active_step_nav_content_id = active_step_nav_content_id
```

Comment:
You can make @active_step_nav_id accessible by setting the `attr_reader` property.